README.md

```diff
@@ -157,6 +170,7 @@ outdated information.
 
 <sup>1</sup> Other machine-readable formats like CSV are supported.
 
+<a name="cliusage"></a>
```

Comment:
I think all of these `a` links can be removed. TOC generator will figure it out. 😉

Reply:
Done 🙌.